packages/react-intl/tests/unit/react-intl.ts

```diff
@@ -1,4 +1,8 @@
 import * as ReactIntl from '../../'
+import * as ts from 'typescript'
+import fs from 'fs'
+import lexer from 'cjs-module-lexer'
```

Comment:
looks like you need to declare this dep in `package.json`, then you can add it to the BUILD file, along with `typescript`

Reply:
Gotcha - I believe 1bc01f2 should have that sorted out. First time using Bazel so not 100% sure if that's correct.